books/refgram/src/logic/default.md

```diff
@@ -1,45 +1,47 @@
 # Default arguments
 
 Some predicates are often used with the same arguments. While it can be
-possible to define new predicates that wraps them and provide this argument,
-thus would lead to multiple words for the same concepts, and might not be
+possible to define new predicates that wrap them and provide this argument,
+this would lead to multiple words for the same concepts, and might not be
 practical for predicates made with some grammar rules such as numbers. Also,
 those common arguments might be specific to some text or speech.
 
 To address the above, Eberban allows the management of __default arguments__,
 which are properties that some arguments must satisfy __only when there are no
 explicit bounds on them__.
 
-Let's take as exemple the predicate __espuateran__, which means
+Let's take as an example the predicate __espuateran__, which means
 
 \\[\text{espuateran}(c,e,a) = [\text{$e$ is the home planet of species $a$}]\\]
 
 We could want that __A__ are humans by default so that we don't have to always
 specify it is the home planet of humans/Earth nor have to define another
 separate predicate.
 
-To do that, we can use the __poie/poia/poio/poiu__ serie in __PO__ to define
-what is this default property: `poia espuateran flan` defines that by default
-the __A__ place of __espuateran__ satisfy the property __flan__ (being a human).
-The __E__ place don't have a default bind, which can be seen as having the
-default bind of satisfying __mai__ (exists).
+To do that, we can use the __poie/poia/poio/poiu__ series in __PO__ to define
```

Comment:
From what I read there is no singular for "series" in English? Very weird.

Reply:
Welcome to English!